qemu/tests/trim_support_test.py

```diff
@@ -48,6 +49,12 @@ def _disk_size_smaller(ori_size):
             return new_size
         return None
 
+    def query_system_events(filter_options):
+        """Query the system events in filter options."""
+        logging.info("Query the system event log.")
+        cmd = 'wevtutil qe system /q:\"%s\" /f:text' % filter_options
+        return session.cmd(cmd).strip()
```

Comment:
Could you show me the regular output of `session.cmd(cmd).strip()` and the exceptional situation?

Reply:
session.cmd(cmd).strip()  should as flowing  if hit disk  corruption for 4k.

04:23:10 ERROR|   Log Name: System
04:23:10 ERROR|   Source: Ntfs
04:23:10 ERROR|   Date: 2021-03-19T03:11:05.1790000Z
04:23:10 ERROR|   Event ID: 55
04:23:10 ERROR|   Task: N/A
04:23:10 ERROR|   Level: Error
04:23:10 ERROR|   Opcode: Info
04:23:10 ERROR|   Keyword: N/A
04:23:10 ERROR|   User: S-1-5-18
04:23:10 ERROR|   User Name: NT AUTHORITY\SYSTEM
04:23:10 ERROR|   Computer: DESKTOP-R1M00EJ
04:23:10 ERROR|   Description:
04:23:10 ERROR| A corruption was discovered in the file system structure on volume D:.
04:23:10 ERROR|
04:23:10 ERROR| A corruption was found in a file system index structure.  The file reference number is 0x9000000000009.  The name of the file is "<unable to determine file name>".  The corrupted index attribute is ":$SDH:$INDEX_ALLOCATION".

and lasted result:
 (1/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1687.15 s)
 (2/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.trim_support_test.q35: PASS (485.09 s)
 (3/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.trim_support_test.with_4k_disk.q35: PASS (325.27 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-03-19T07.04-c7f17d1/results.html
JOB TIME   : 2498.94 s